awsagentprovider/src/main/java/software/amazon/opentelemetry/javaagent/providers/AwsMetricAttributeGenerator.java

```diff
@@ -493,11 +492,6 @@ private static void setRemoteResourceTypeAndIdentifier(SpanData span, Attributes
                 Optional.ofNullable(escapeDelimiters(span.getAttributes().get(AWS_SECRET_ARN))));
         cloudformationPrimaryIdentifier =
             Optional.ofNullable(escapeDelimiters(span.getAttributes().get(AWS_SECRET_ARN)));
-      } else if (isKeyPresent(span, AWS_LAMBDA_NAME)) {
-        remoteResourceType = Optional.of(NORMALIZED_LAMBDA_SERVICE_NAME + "::Function");
-        remoteResourceIdentifier =
-            getLambdaResourceNameFromAribitraryName(
```

Comment:
Minor: [This leftover function](https://github.com/aws-observability/aws-otel-java-instrumentation/pull/907/files#diff-4fd9c64d196773397186abbbded179e6491214eeb417177f4b40cc0ac63f9d19R517) can now be removed